packages/manager/modules/anthos/src/dashboard/host/remove/translations/Messages_fr_FR.json

```
@@ -0,0 +1,8 @@
+{
+  "anthos_dashboard_remove_host_title": "Supprimer l'hôte",
+  "anthos_dashboard_remove_host_description": "Cette machine sera immédiatement supprimée, et vous ne serez plus facturé au dela de sa date de fin d'engagements ( le {{expiryDate}}).",
+  "anthos_dashboard_remove_host_warning": "A La suppression entrainera l'effacement définitif de toutes les données correspondantes.",
```

Doesn't sound correct. Content has been validated?

```suggestion
  "anthos_dashboard_remove_host_warning": "La suppression entrainera l'effacement définitif de toutes les données correspondantes.",
```

---

Yeah... It was a typo. Fixed now. Thanks!